docs/src/e2e.md

```diff
@@ -127,3 +127,6 @@ and the following suite of E2E tests are performed on that cluster:
      * Declarative creation of temporary tablespaces
      * Backup / recovery from object storage
      * Backup / recovery from volume snapshots
+
+* **Publication and Subscription:**
```

Comment:
IMO this belongs in the Replication grouping, we don't need a top-level grouping for it